spec/lib/rbac/filterer_spec.rb

```diff
@@ -212,6 +212,60 @@ def combine_filtered_ids(user_filtered_ids, belongsto_filtered_ids, managed_filt
         tagged_group.save!
       end
 
+      context 'searching for instances of IsoImages' do
+        let!(:iso_image) { FactoryBot.create_list(:iso_image, 2).first }
```

Comment:
I'm curious why `create_list` + `.first` is used here instead of just `create`. Is it just so that there's more than one in the database?

Reply:
I create two `IsoImage`s and take first one to give it a tag. Then I test that admin sees both and  user with limited access sees just that one with tag. So yeah I need more than one in the database :)

Reply:
@ZitaNemeckova Ok, just wanted to make sure, thanks!